sheet_values.py

```diff
@@ -25,53 +27,55 @@ def getItemStrings():
 
     # Remove object plural strings
     itemSTR = itemSTR[~itemSTR['label'].str.endswith('_pl').fillna('')]
-    
+
     # Strip label to ID only and remove yucky chars from name string
     itemSTR = stripString(itemSTR)
-    
+
     # Cast label to int64
     itemSTR = itemSTR.astype({'label': 'int64'})
-    
+
     # Rename columns
     itemSTR.rename(columns={'text': 'Name_Items', 'label': 'Internal ID'}, inplace=True)
 
     return itemSTR
 
+
 def getClothingStrings():
     path = 'data/game_parsed/acnh-message-usen/String_USen/Outfit/GroupName'
     clothing = ['STR_OutfitGroupName_Accessory.csv', 'STR_OutfitGroupName_Bag.csv', 'STR_OutfitGroupName_Bottoms.csv', 'STR_OutfitGroupName_Cap.csv', 'STR_OutfitGroupName_Helmet.csv', 'STR_OutfitGroupName_MarineSuit.csv', 'STR_OutfitGroupName_OnePiece.csv', 'STR_OutfitGroupName_Shoes.csv', 'STR_OutfitGroupName_Socks.csv', 'STR_OutfitGroupName_Tops.csv']
 
     # Get clothing name strings
     clothingSTR = pd.concat([pd.read_csv(f'{path}/{file}', dtype=object) for file in clothing])
-    
+
     # Strip label to ID only and remove yucky chars from name string
     clothingSTR = stripString(clothingSTR)
-    
+
     # Cast label to int64
     clothingSTR = clothingSTR.astype({'label': 'int64'})
-    
+
     # Rename column 'text' to 'Name'
-    clothingSTR.rename(columns={'text': 'Name_Clothing', 'label' : 'ClothGroup ID'}, inplace=True)
-    
+    clothingSTR.rename(columns={'text': 'Name_Clothing', 'label': 'ClothGroup ID'}, inplace=True)
+
     return clothingSTR
 
+
 def filenameToClothGroup(item, sequence):
-    options = [label for index, label in sequence.items() if item.startswith(label)]
-    if options == []:
+    options = [label for label in sequence.values() if item.startswith(label)]
```

Comment:
note to self, look up difference between sequence.items() and sequence.values() later.

Reply:
There are 3 + 1 ways to iterate over a dictionary

1. Iterate over keys (default)

`for key in my_dict:`

which is equivalent to:

`for key in my_dict.keys():`

2. Iterate over values

`for values in my_dict.values():`

3. Iterate over keys and values

`for key, values in my_dict.items():`

In your case, since you weren't using the keys, there was no need to include those.